### DISCUSSION TITLE
Welcome to Mdisk_downloader-main Discussions!

### DISCUSSION THREAD

**Discussion by Devansh-Bots** [Devansh-Bots/Mdisk_downloader-main#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Devansh-Bots** [comment]
ha vai puchho ab questions..

**Comment by noob-mukesh** [comment]
update kro code sir not working

**Reply by Devansh-Bots** [reply]
and how to do that?
